src/utils/updatePackageVersions.js

```diff
@@ -0,0 +1,24 @@
+//@flow
+import Project from '../Project';
+import Workspace from '../Workspace';
+import runWorkspaceTasks from './runWorkspaceTasks';
+
+export default async function updatePackageVersions(versions: {[name: string]: string}, opts: { cwd?: string } = {}) {
+  const cwd = opts.cwd || process.cwd();
+  const project = await Project.init(cwd);
+  const workspaces = await project.getWorkspaces();
+
+  for (let workspace of workspaces) {
+    const pkg = workspace.pkg;
+    const promises = [];
+
+    for (let depName of Object.keys(versions)) {
+      const depType = pkg.getDependencyType(depName);
+      if (!depType) continue;
+
+      promises.push(pkg.updateDependencyVersionRange(depName, depType, versions[depName]));
```

Comment:
This should probably be:

```js
'^' + versions[depName]
```

That way the version ranges that we're publishing can match a wider range

Reply:
Oh, in testing, I was adding the hat in myself.